packages/jsii-pacmak/lib/targets/go/documentation.ts

```diff
@@ -49,13 +46,19 @@ export class Documentation {
     const stability = docs.stability;
     if (stability && this.shouldMentionStability(docs)) {
       if (docs.deprecated) {
-        this.code.line(`// Deprecated: ${docs.deprecationReason}`);
+        this.emitComment(`Deprecated: ${docs.deprecationReason}`);
       } else {
-        this.code.line(`// ${this.code.toPascalCase(stability)}.`);
+        this.emitComment(`${this.code.toPascalCase(stability)}.`);
       }
     }
   }
 
+  private emitComment(text = '') {
+    for (const line of text.split('\n')) {
+      this.code.line(`// ${line}`);
```

Comment:
```suggestion
      this.code.line(`// ${line}`.trim());
```

Reply:
I've added this but then the snapshots actually changed much more dramatically so I decided to revert. I'll add.